model/fn-execution/src/main/resources/org/apache/beam/model/fnexecution/v1/standard_coders.yaml

```diff
@@ -455,6 +455,17 @@ examples:
 
 ---
 
+coder:
+  urn: "beam:coder:row:v1"
+  # f_float: float32
```

Comment:
why f_float instead of float or float32?

Reply:
f_float here is the name of a field in the schema. We could call it "float" or "float32" instead, but it's not that important. I just followed the f_<type> convention used elsewhere.